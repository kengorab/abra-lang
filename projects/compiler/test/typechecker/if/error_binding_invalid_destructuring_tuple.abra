val arr = [1, 2, 3]
if arr[0] |(a, b)| {

}