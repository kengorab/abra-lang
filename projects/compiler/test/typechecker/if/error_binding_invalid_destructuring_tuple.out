Error at %FILE_NAME%:2:12
Invalid destructuring
  |  if arr[0] |(a, b)| {
                ^
A value of type 'Int' cannot be destructured as a tuple
